CHANGES.txt

```diff
@@ -1,3 +1,13 @@
+9.6.1 (Nov 3, 2023)
+- Added support for Flag Sets on the SDK, which enables grouping feature flags and interacting with the group rather than individually (more details in our documentation):
+  - Added new variations of the get treatment methods to support evaluating flags in given flag set/s.
+    - getTreatmentsByFlagSet and getTreatmentsByFlagSets
```

Comment:
please update to `get_treatments_by_flag_set` etc etc according Python API

Reply:
will do